packages/expo-cli/bin/expo.js

```diff
@@ -7,7 +7,9 @@ var ver = process.versions.node;
 ver = ver.split('-')[0]; // explode and truncate tag from version
 
 if (semver.satisfies(ver, '>=6.0.0')) {
-  require('source-map-support').install();
+  if (process.env.EXPO_DEBUG === 'true') {
```

Comment:
Sometimes people will set EXPO_DEBUG=1, need to use something like getenv here: https://github.com/expo/expo-cli/blob/04f9572e091ce96c4b490afe94b8c5150de8fd25/packages/xdl/src/project/Doctor.js#L444